roxctl/image/check/check.go

```diff
@@ -85,6 +85,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	pkgUtils.Must(c.MarkFlagRequired("image"))
 	c.Flags().IntVarP(&imageCheckCmd.retryDelay, "retry-delay", "d", 3, "set time to wait between retries in seconds.")
 	c.Flags().IntVarP(&imageCheckCmd.retryCount, "retries", "r", 3, "number of retries before exiting as error.")
+	c.Flags().BoolVarP(&imageCheckCmd.force, "force", "f", false, "the --force flag ignores Central's cache for the check and forces a fresh re-pull from Scanner")
```

Comment:
Wording: cache for the image scan and instead forces a refresh from Scanner. Maybe something like that. Just describing which cache it's ignoring

Reply:
Fixed in latest commit